src/lib/components/TextContext/TextContent.jsx

```diff
@@ -514,19 +483,79 @@ TextContent.propTypes = {
     "style10",
     "style11",
     "style12",
+    "style13",
   ]),
   size: PropTypes.oneOf(["large", "medium", "small"]),
   rating: PropTypes.number,
   textGenre: PropTypes.string,
   title: PropTypes.string,
   description: PropTypes.string,
   heading: PropTypes.string,
-  badgeText: PropTypes.string,
+  badgeText: PropTypes.array,

```

Comment:
Can you rename badgeText prop to more descriptive and plural string as it denotes array?